config/schema/document_types/international_development_fund.json

```diff
@@ -33,8 +33,8 @@
         "value": "burma"
       },
       {
-        "label": "Democratic Republic of Congo",
-        "value": "democratic-republic-of-congo"
```

Comment:
Let's keep this value the same for now, so that we don't have to republish everything (if it's changed now, the search won't work).